macros/models/_sector/dex/uniswap_compatible_trades.sql

```diff
@@ -73,7 +73,7 @@ WITH dexs AS
         , CASE WHEN amount0 < INT256 '0' THEN f.token0 ELSE f.token1 END AS token_bought_address
         , CASE WHEN amount0 < INT256 '0' THEN f.token1 ELSE f.token0 END AS token_sold_address
         , t.contract_address as project_contract_address
-        , f.fee
+        --, f.fee
```

Comment:
Do you think we may further extend the macro's compatibility by adding optional columns in the signature? Like default `optional_columns = ['f.fee']`.

Reply:
Sure, good idea. My note of fee column not being used down the line still stands though, but worth having a safety net.